ted_sws/master_data_registry/services/rdf_fragment_processor.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python3
+
+# rdf_fragment_processor.py
+# Date:  29.07.2022
+# Author: Stratulat Ștefan
+# Email: stefan.stratulat1997@gmail.com
+
+"""
+
+"""
+import pathlib
+from string import Template
+from typing import List
+
+from rdflib import Graph
+
+from ted_sws.master_data_registry.model.rdf_fragment import RDFFragment
+from ted_sws.master_data_registry.resources import RDF_FRAGMENT_BY_URI_SPARQL_QUERY_TEMPLATE_PATH, \
+    TRIPLES_BY_CET_URI_SPARQL_QUERY_TEMPLATE_PATH
+
+DEFAULT_RDF_FILE_FORMAT = "n3"
+
+
+def get_rdf_fragment_by_cet_uri_from_string(rdf_file_content: str, cet_uri: str,
```

Comment:
the logic in this function shall be implemented as a new class in the data_manager sparql_endpoint.py module.

the class shall be 

```
class SPARQLFileEndpoint(TripleStoreEndpointABC):
    ...
```
if necessary you may also create something to parse and query an arbitrary string, but I don't think we need it. 

```
class SPARQLStringEndpoint(TripleStoreEndpointABC):
    ...
``` 


![image](https://user-images.githubusercontent.com/13567133/182693950-0c05aadb-36cd-4cc9-a1e1-6cc80f0ebbd6.png)